src/VirtoCommerce.PriceExportImportModule.Web/Scripts/module.js

```diff
@@ -7,15 +7,135 @@ if (AppDependencies !== undefined) {
 
 angular.module(moduleName, [])
     .run(
-        ['virtoCommerce.featureManagerSubscriber', 'platformWebApp.widgetService', 'platformWebApp.authService',
-            function (featureManagerSubscriber, widgetService, authService) {
+        ['virtoCommerce.featureManagerSubscriber', 'platformWebApp.toolbarService', 'platformWebApp.bladeNavigationService', 'platformWebApp.dialogService', 'platformWebApp.settings',
+            function (featureManagerSubscriber, toolbarService, bladeNavigationService, dialogService, settings) {
                 featureManagerSubscriber.onLoginStatusChanged('PriceExportImport', () => {
 
-                    widgetService.registerWidget({
-                        isVisible: function (blade) { return blade.controller === 'virtoCommerce.pricingModule.pricelistDetailController' && !blade.isNew && authService.checkPermission('pricing:read'); },
-                        controller: 'virtoCommerce.priceExportImportModule.pricesWidgetController',
-                        template: 'Modules/$(VirtoCommerce.PriceExportImport)/Scripts/widgets/pricesWidget.tpl.html'
-                    }, 'pricelistDetail');
+                    toolbarService.register({
+                        name: "platform.commands.export", icon: 'fa fa-upload',
+                        executeMethod: function (blade) {
+                            const csvExportDelimiter = ';';
+                            const csvExportProvider = 'CsvExportProvider';
+                            const csvPropertyInfo = [
+                                getCsvPropertyInfo("Code", "SKU"),
```

Comment:
It is better smth like this "createCsvPropertyInfo". It is fabric. Not a getter.

Reply:
done